src/components/ReceiptImage.tsx

```diff
@@ -115,10 +116,11 @@ function ReceiptImage({
             <ThumbnailImage
                 previewSourceURL={source ?? ''}
                 style={[styles.w100, styles.h100]}
-                isAuthTokenRequired
+                isAuthTokenRequired={isAuthTokenRequired ?? false}
```

Comment:
Could you quickly comment on the changes related to the auth token? (this line, but also some other lines in the whole PR diff)

Does adjusting the thumbnail positioning have some relation with auth tokens? I'm trying to ensure we got conflict resolution right.

Reply:
Sorry, I forgot. This is definitely a sign we should wrap this up.